sdk/script/management-sdk.ts

```diff
@@ -58,18 +44,24 @@ function urlEncode(strings: string[], ...values: string[]): string {
     return result;
 }
 
-export class AccountManager {
+class AccountManager {
+    public static PERMISSIONS = {
+        OWNER: "Owner",
+        COLLABORATOR: "Collaborator"
+    };
     public static SERVER_URL = "https://codepush-management.azurewebsites.net";
 
     private static API_VERSION: number = 2;
 
     private _accessKey: string;
     private _serverUrl: string;
-    private _userAgent: string;
+    private _customHeaders: Headers;
+
+    constructor(accessKey: string, customHeaders?: Headers, serverUrl?: string) {
+        if (!accessKey) throw new Error("An access key must be specified.");
 
-    constructor(accessKey: string, userAgent?: string, serverUrl?: string) {
         this._accessKey = accessKey;
-        this._userAgent = userAgent;
+        this._customHeaders = customHeaders;
```

Comment:
Would there be a value in also having a default header for the SDK, and then merge the supplied headers with ours?